update-bot/config/templates.json

```diff
@@ -36,8 +36,8 @@
       },
       "Denkmalliste Sachsen-Anhalt Tabellenzeile": {
           "id": "ID",
-          "id_check": "\\d{8}",
-          "id_check_description": "achtstellige Nummer"
+          "id_check": "\\d{3}\\s*\\d{5}",
```

Comment:
It is intended because the description of the number in Wikipedia is "Achtstellige Nummer" without specifying the formatting.